integration/specs/avatarMenu-1.spec.js

```diff
@@ -0,0 +1,116 @@
+    });
+    it('should put the menu button focused when the menu is opened and press ESC', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        avatarMenu.click();
+        browser.keys(ESCAPE_KEY);
+        expect(avatarMenu.hasFocusButton()).toBe(true);
+    });
+    it('should open the menu when the button is focused and press Space', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        avatarMenu.click();
+        browser.keys(ESCAPE_KEY);
+        browser.keys(SPACE_KEY);
+        expect(avatarMenu.isOpen()).toBe(true);
+    });
+    it('should open the menu when the button is focused and press Enter', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        avatarMenu.click();
+        browser.keys(ESCAPE_KEY);
+        browser.keys(ENTER_KEY);
+        expect(avatarMenu.isOpen()).toBe(true);
+    });
+    it('should set the focus to the first menu item when open the menu', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        const menuItem = avatarMenu.getItem(0);
+        avatarMenu.click();
+        expect(menuItem.hasFocus()).toBe(true);
+    });
+    it('should move to the next item when press arrow down', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        const menuItem = avatarMenu.getItem(1);
+        avatarMenu.click();
+        browser.keys(ARROW_DOWN_KEY);
+        expect(menuItem.hasFocus()).toBe(true);
+    });
+    it('should move to the previous item when press arrow up', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        const menuItemOne = avatarMenu.getItem(0);
+        const menuItemTwo = avatarMenu.getItem(1);
+        avatarMenu.click();
+        menuItemTwo.hover();
+        browser.keys(ARROW_UP_KEY);
+        expect(menuItemOne.hasFocus()).toBe(true);
+    });
+    it('should move to the first item when last item is active and press arrow down', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        const firstMenuItem = avatarMenu.getItem(0);
+        const lastMenuItem = avatarMenu.getItem(1);
+        avatarMenu.click();
+        lastMenuItem.hover();
+        browser.keys(ARROW_DOWN_KEY);
+        expect(firstMenuItem.hasFocus()).toBe(true);
+    });
+    it('should move to the last item when first item is active and press arrow up', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        const firstMenuItem = avatarMenu.getItem(0);
+        const lastMenuItem = avatarMenu.getItem(1);
+        avatarMenu.click();
+        firstMenuItem.hover();
+        browser.keys(ARROW_UP_KEY);
+        expect(lastMenuItem.hasFocus()).toBe(true);
+    });
+    it('should close the menu when the menu is opened and press the key "tab"', () => {
+        const avatarMenu = new PageAvatarMenu(AVATAR_MENU);
+        avatarMenu.click();
+        browser.keys(TAB_KEY);
+        expect(avatarMenu.isOpen()).toBe(false);
```

Comment:
add:
```expect(avatarMenu.hasFocusButton()).toBe(false);```
and then update the name:
should close the menu and not have focus in the trigger menu button when the menu is opened and press the tab key